tier2_standards/citation_information.md

```diff
@@ -4,14 +4,51 @@ This is one of the tier 2 standards. See full list [on the overview page](README
 
 ## Short description
 
+We require that the project documentation or repository contains a description of how to properly cite the project in publications and/or other software.
 
 ## Importance of this standard
 
+Unlike journal articles, software repositories do not have a clear title page or list of authors.  Providing a citation description makes it easier for (and encourages) users to correctly cite your software in publications and acknowledgements.
 
 ## Options for this standard
 
+We require that teams choose at least one of the following options:
+
+- Include a `CITATION` file in the main directory of the repository that contains an appropriate description.
+- Include an appropriate description in the repository's main `README` file.
+- Include an appropriate description in the project's main documentation source (e.g. JDox, ReadTheDocs, etc.).
+
+Though not required, another great way to make your software easily citable is to register the project with [Zenodo](https://zenodo.org/).  Zenodo is an open-access tool that allows users to upload software (and any accompanying data, reports, presentations, etc.) and receive a digital object identifier (DOI) for each submission.  For more information, visit https://about.zenodo.org/, and to sign up, visit https://zenodo.org/signup/.
 
 ## How to apply this standard
```

Comment:
Is bibtex the go-to way of citing stuff in papers? I want to make sure we're giving the teams everything they need. Do we also need something like a one-line citation? (I don't know how citing stuff like this works, so if bibtex is the normal thing to do then that's fine).

Reply:
Good question.  I hadn't really given it much thought since I took it directly from Style Guides.  I can tweak this example a bit so that it includes an option to add a one-line citation.